mobile-widgets/uploadDiveLogsDE.cpp

```diff
@@ -257,6 +261,8 @@ void uploadDiveLogsDE::updateProgress(qint64 current, qint64 total)
 
 void uploadDiveLogsDE::uploadFinished()
```

Comment:
the declaration (and the use in the previous hunk) have two arguments, this has no arguments. That seems wrong? Or am I missing some overloading here?

Reply:
nope, "uploadFinish(a, b)" is a signal, without implementation, and uploadFinished() is a SLOT with implementation. Agree naming could have been more different, but they do the same thing, so I kept them close.